### ISSUE TITLE
Seems to require Qt4

### ISSUE THREAD

**Issue by MiBengtsson** [skypjack/qsapecng#2]
Are there plans to move to Qt5?

**Comment by skypjack** [comment]
Unfortunately, I've been looking for funds for a long time to maintain/update this project but without success.
I'd be more than willing to work on this though. On the other hand, I don't have enough bandwidth to do it in my spare time.
So, unless companies or individuals interested in it show up, I think it won't be upgraded to Qt5 soon. I'm sorry.

**Comment by weatherhead99** [comment]
I have been looking for a tool like this for a long time (considered writing my own before finding yours, with a very nice GUI!)

So, I started quickly porting to Qt5. Without that much work I have managed to get it to build against Qt5, see branch here  [https://github.com/weatherhead99/qsapecng/tree/qt5_port](https://github.com/weatherhead99/qsapecng/tree/qt5_port)

it requires the qt5 port of QtPropertyBrowser, which I have forked here (and added a quick CMake build system to)
[https://github.com/weatherhead99/QtPropertyBrowser/tree/cmake](https://github.com/weatherhead99/QtPropertyBrowser/tree/cmake).

I would be interested to help out a bit with better porting / improvements, as this tool is very valuable to me! But I don't have a vast amount of effort to put in at the moment.

**Comment by skypjack** [comment]
Interesting and thanks for getting the time to look into this.
Wouldn't it worth porting it directly to Qt6 though? Is there any reason for which it shouldn't work?

**Comment by weatherhead99** [comment]
no I don't think there's any reason it should not work, I just did this because I'm not familiar with Qt6 yet.

Actually probably the biggest obstacle would be porting QtPropertyBrowser to Qt6. It saved me most of the work to find that someone else had already ported this to Qt5.

I think the port should be pretty simple actually.  It looks like the changes from 5->6 are somewhat smaller than 4->5 actually.

I'm not sure if Qwt has a qt6 release yet either...

**Comment by skypjack** [comment]
Makes sense. What about turning your work in a PR onto a dedicate branch?
We can then refine/improve the port to Qt5 in our free time, a little at a time.
